### ISSUE TITLE
Add guardrail against catastrophic build directory removal

### ISSUE THREAD

**Issue by erxclau** [MichiganDaily/sink#67]
GitHub deployment script runs `rm -rf` on a specified build directory. This could be potentially catastrophic if someone specifies a non-build directory. For caution, ensure that the build directory specified is within the present working directory.

**Comment by erxclau** [comment]
This is no longer relevant after #73 changes which creates a temporary worktree directory which only copies content from the build directory.
